apps/blockchain/lib/blockchain/chain.ex

```diff
@@ -181,14 +190,20 @@ defmodule Blockchain.Chain do
     }
   end
 
-  @doc """
-  Checks if this account should be included in the genisis block
-
-  Note: Ropsten's builtin accounts have balances
-  # https://github.com/poanetwork/mana/blob/master/chains/ropsten.json#L1583
-  """
-  defp include_in_genisis?({raw_address, info}) do
-    !Map.has_key?(info, "builtin") || Map.has_key?(info, "balance")
+  defp get_accounts(json_accounts) do
+    accounts =
+      Enum.reduce(json_accounts, [], fn json_account = {_address, info}, acc ->
+        account =
+          if is_nil(info["builtin"]) do
+            get_account(json_account)
+          else
+            get_builtin_account(json_account)
+          end
+
+        acc ++ [account]
```

Comment:
Would it be better to prepend to the list instead of adding it to the end, `[account | acc]`? Not sure how important it is, but it should be slightly more performant. The only downside is that it would invert the order of elements in the list since it's prepending them.